src/main/java/com/thed/zephyr/jenkins/reporter/ZeeReporter.java

```diff
@@ -197,9 +183,10 @@ public boolean perform(final Run build, final TaskListener listener) throws IOEx
             }
 
             for(String xmlFilePath : xmlFiles) {
-                dataMapList.addAll(genericParserXML(xmlFilePath, parserTemplateArr[Integer.valueOf(String.valueOf(zephyrConfigModel.getParserIndex()))]));
+                dataMapList.addAll(genericParserXML(xmlFilePath, zephyrConfigModel.getJsonParserTemplate()));
```

Comment:
where required java comments and docs will helpful
we cannot add log4j logging and all is it on the jenkins plugin side?

Reply:
Sure, and yes we can add log4j logging.